core-library/src/main/java/org/silverpeas/core/persistence/jcr/AbstractJcrConverter.java

```diff
@@ -238,16 +238,18 @@ protected int getIntProperty(Node node, String propertyName) throws ValueFormatE
    *
    * @param node the node whose property is required.
    * @param propertyName the name of the property required.
+   * @param defaultValueIfNull the default value of the value does not exist
    * @return the boolean value of the property - false if the property doesn't exist.
    * @throws RepositoryException on error
    * @throws ValueFormatException on error
    */
-  protected boolean getBooleanProperty(Node node, String propertyName) throws ValueFormatException,
-      RepositoryException {
+  protected boolean getBooleanProperty(Node node, String propertyName,
+      final boolean defaultValueIfNull) throws ValueFormatException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'javax.jcr.ValueFormatException' which is a subclass of 'javax.jcr.RepositoryException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)